examples/template.yml

```diff
@@ -0,0 +1,80 @@
+# https://github.com/awslabs/aws-cloudformation-templates/blob/master/community/services/Lambda/LambdaSample.yaml
```

Comment:
Looks like a linting error was surfaced in https://github.com/ScottBrenner/cfn-lint-action/runs/2373860426?check_suite_focus=true#step:5:80
> Error: Error running command: cfn-lint -t ./template.yml. Returned error is: You have Errors in your Cloud Formation

but the `npm test` job still passed, should it fail in this situation? Not clear what the offending linting errors are

Reply:
Good point out @ScottBrenner. This is actually meant to happen 👍 I have a unit test that checks and makes sure I am handling the failure successfully. I want to make sure that we do actually fail the build if there are errors and this is being tests by what you just pointed out 💯